lab_3_keywords_textrank/main.py

```diff
@@ -251,7 +302,12 @@ def is_incidental(self, vertex1: int, vertex2: int) -> int:
                 1 if vertices are incidental, otherwise 0
         If either of vertices is not present in the graph, -1 is returned
         """
-        pass
+        if vertex1 not in self._vertexes or vertex2 not in self._vertexes:
+            return -1
+        answer = self._matrix[self._vertexes[vertex1]][self._vertexes[vertex2]]
+        if isinstance(answer, int):
```

Comment:
what another type could it be and why?

Reply:
i don't know, but it gives me an error in "check types" saying that "returned type any when expected type int". this check allows me to overpass this error

Reply:
maybe it might be "none" if it doesn't exist in `self._matrix`? but it cannot "not exist" there

Reply:
It is strange. You initialize all as integers in matrix